cate/webapi/start.py

```diff
@@ -98,28 +98,33 @@ def service_factory(application):
 # }
 
 def create_application(user_root_path: str = None):
-    application = Application([
-        ('/_static/(.*)', StaticFileHandler, {'path': FigureManagerWebAgg.get_static_file_path()}),
-        ('/mpl.js', MplJavaScriptHandler),
-
-        (url_pattern('/mpl/download/{{base_dir}}/{{figure_id}}/{{format_name}}'), MplDownloadHandler),
-        (url_pattern('/mpl/figures/{{base_dir}}/{{figure_id}}'), MplWebSocketHandler),
+    default_url_root = "/"
+    # replace default url_root with /user/username/ if running in Cate Hub context.
+    url_root = os.environ.get("JUPYTERHUB_SERVICE_PREFIX", default_url_root)
+    if url_root is not default_url_root:
+        print(f"warning: detected jupyterhub environment variable JUPYTERHUB_SERVICE_PREFIX "
```

Comment:
Why "warning"? Is it a bad thing that this variable is set? I'd prefer "info" or nothing.

Reply:
Nothing is not good here as environment is detected automatically, info would be good but there is another warning in the module which does similar thing, just retaining it for consistency.